edx/frontend-app-gradebook
# fix: use SITE_NAME env var for index.html title

We currently have a hard-coded “edX” string in Gradebook’s index.html file.  This replaces that string with the value of the SITE_NAME environment variable, which is set at build time from .env.development for dev, and from the build process at production build-time.  

JIRA: https://openedx.atlassian.net/browse/CENG-6

**What changed?**

This updates index.html to use SITE_NAME, and sets the .env.development default to 'localhost', and the .env default to an empty string.

**Developer Checklist**

**Testing Instructions**

If you start your local development server for gradebook and load it in the browser, the title bar should default to "Gradebook | localhost" in the source of the HTML file.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:



FYI: @edx/masters-devs-gta
